modules/ROOT/pages/connected-apps-end-users.adoc

```diff
@@ -0,0 +1,19 @@
+= Connected Apps for End Users
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+End users can delegate API access and log in to third-party applications using their Anypoint Platform credentials.
+They can can authorize access to their account when the following message appears:
+
+.The *Authorize App* interface enables end users to see which organization is requesting access to data and how the data is used. After the end user clicks *Grant Access*, the app is authorized to access the data indicated in the *Authorize App* interface.
+image::connected-apps-authorize.png[The interface the user sees before authorizing an app to use their data, 60%,60%]
+
+
+End users can manage third-party apps that they have authorized in their profile settings. They can view authorized apps in their profiles by clicking on their profile icon in the top right corner and selecting *Profile*. The following screenshot illustrates how authorized connected apps are shown within a user’s profile page:
```

Comment:
Line 13 mentions "the following screen shot" but I don't see an image in the vicinity. ?

Reply:
Same as above.